tests/unit_tests/test_wave_06.py

```diff
@@ -76,7 +76,12 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result is True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory 
+    assert item_c in jesse.inventory
```

Comment:
Great start! It would be a good idea to make sure the other items that we expect to be in these inventories are there as well!